website/docs/export.md

```diff
@@ -209,6 +211,90 @@ Dapr: https://dapr.io/
     2023/07/18 20:37:20 main.ExportMsg{ID:"2023-07-18T20:37:19Z", Details:map[string]interface {}{"missing_labels":[]interface {}{"test"}}, EventType:"violation_audited", Group:"constraints.gatekeeper.sh", Version:"v1beta1", Kind:"K8sRequiredLabels", Name:"pod-must-have-test", Namespace:"", Message:"you must provide labels: {\"test\"}", EnforcementAction:"deny", ConstraintAnnotations:map[string]string(nil), ResourceGroup:"", ResourceAPIVersion:"v1", ResourceKind:"Pod", ResourceNamespace:"nginx", ResourceName:"nginx-deployment-58899467f5-j85bs", ResourceLabels:map[string]string{"app":"nginx", "owner":"admin", "pod-template-hash":"58899467f5"}}
     ```
 
+### Quick start with exporting violations on node storage using Disk driver via emptyDir
+
+#### Configure Gatekeeper with Export enabled to Disk
+
+1. Deploy Gatekeeper with disk export configurations.
+
+    ```shell
+    helm upgrade --install gatekeeper gatekeeper/gatekeeper --namespace gatekeeper-system \
+    --set enableViolationExport=true \
+    --set audit.connection=audit-connection \
+    --set audit.channel=audit-channel \
+    --set audit.exportConfig.maxAuditResults=3 \
+    --set exportBackend=disk \
+    --set audit.exportSidecar.image=ghcr.io/open-policy-agent/fake-reader:latest \
```

Comment:
since this is a default value, should this be removed assuming users have updated their values.yaml or add a comment here to `--set audit.exportSidecar.image` to override the default?

Reply:
+1, remoded these values.